### ISSUE TITLE
What is the most accessible way to get ulid in Docker image?

### ISSUE THREAD

**Issue by gajus** [alizain/ulid#53]
Installing the entire Node.js as a dependency is ... inefficient to say the least.

None of the Python libraries seem to provide CLI interface.

Is there a recommended program?

I need ulid ID as part of a build process.

**Comment by gajus** [comment]
A fairly reasonable one-liner:

```bash
apt-get update -y && apt-get install -y python-pip && pip install python-ulid && python -c 'from ulid import ULID; print ULID.new().str'

```

**Comment by alizain** [comment]
You might have better luck with some one of the two `go` implementations. They should be available as static binaries with 0 dependencies!

**Comment by gajus** [comment]
Well, what would be the one-liner for those?

**Comment by alizain** [comment]
I have no idea, you'll have to check with those specific projects.

**Comment by gajus** [comment]
FYI For whatever reason ulid install failed.

```bash
$ apk add --update nodejs nodejs-npm && npm install -g ulid && export RELEASE_ID=$(ulid)
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
(1/8) Installing libcrypto1.0 (1.0.2k-r0)
(2/8) Installing libgcc (6.3.0-r4)
(3/8) Installing http-parser (2.7.1-r1)
(4/8) Installing libssl1.0 (1.0.2k-r0)
(5/8) Installing libstdc++ (6.3.0-r4)
(6/8) Installing libuv (1.11.0-r1)
(7/8) Installing nodejs (6.10.3-r1)
(8/8) Installing nodejs-npm (6.10.3-r1)
Executing busybox-1.26.2-r7.trigger
OK: 43 MiB in 20 packages
/usr/bin/ulid -> /usr/lib/node_modules/ulid/bin/cli.js
/usr/lib
`-- ulid@2.2.0 

module.js:471
    throw err;
    ^

```

**Comment by alizain** [comment]
Oh dear, what a silly bug. I'll have it cleared in a moment.

**Comment by alizain** [comment]
Just released v2.2.1, let me know if it solves your issues!

**Comment by alizain** [comment]
A quick note - running `ulid` on the command line will not output a newline character.

**Comment by gajus** [comment]
> Just released v2.2.1, let me know if it solves your issues!

It does.
